Payments/Operation/02_ManagePaymentsAndCaptures.md

```diff
@@ -0,0 +1,103 @@
+
+#### Prerequisites
+
+- At least one connection has been created, see [Create a connection](../Integration/01_ManageConnections.md#create-a-connection).
+- At least one payment and capture transaction has been created.
+- It is recommended that you add the *Status ID* column to your view so that you are able to sort or filter for specific statuses. 
+
+#### Procedure
+
+*Payments > Payments and Captures > Tab LIST*  
+
+![Payment and capture transactions](../../Assets/Screenshots/Payments/PaymentsCaptures/LISTPaymentsCaptures.png "[Payment and capture transactions]")  
+
+
+1. Check the payment and capture transaction status in the *Status* column. The status displays the current stage in the payment process. You can use the status ID prefixed below to filter the list. The following statuses are available:   
+    - **1 - Preparing**  
+        The transaction has been created, but is not yet transferred.
+    - **2 - Unconfirmed**   
+        A refund is reserved but not yet released by an Actindo user. This status is relevant for some functions only, for example, the cancellations and chargebacks.
+    - **3 - Success**  
+        Actindo has got the response from the service payment provider, that the transfer was successfull.
+    - **4 - Failure**   
+        The payment service provider has received the request, but it cannot process it. If available for the connection, the *Status information* field provides further information.
+    - **5 - Error**   
+       Errors have been occurred during transfer.
+    - **6 - Void**   
+       The transaction has been voided, see [Void payment and capture transaction](#void-payment-and-payment-and-capture-transaction).
+    
+2. Click on a transaction to see the details.   
+    The sub-tabs of the transaction are opened.   
+
+    *Payments > Payments and Captures > Tab LIST > Select a payment and capture transaction*   
+
+    ![Payment and capture attributes and logs](../../Assets/Screenshots/Payments/PaymentsCaptures/CheckAttributes.png "[Payment and capture attributes and logs]")   
+
+3. Check the attributes and logs of the transaction. For detailed information, see the following:
+     - [LIST &ndash; Attributes](../UserInterface/03_ListPaymentsAndCaptures.md#payments-and-captures-–-attributes)
+     - [LIST &ndash; Logs](../UserInterface/03_ListPaymentsAndCaptures.md#payments-and-captures-–-logs)
+4. Return to the LIST and decide how to proceed with the transaction.
+
+
+
+## Refund payment and capture transaction
+
+After you have checked a payment and capture transaction that could not be processed by the workflow, you can refund the payment and capture transaction manually, so that the order process can be cancelled and the customer can get his or her money back.<!---ist das richtig-->
```

Comment:
```suggestion
After you have checked a payment and capture transaction that could not be processed by the workflow, you can refund the payment and capture the transaction manually, so that the order process can be cancelled and the customer can get his or her money back.<!---ist das richtig-->
```

Reply:
done